.github/ISSUE_TEMPLATE

```diff
@@ -1,6 +1,11 @@
 <!--
 Please make sure your issue title matchs the Go convention; a summary
 of the problem, prefixed by the primary affected package.
+
+If the issue is a question about usage or similar, please ask at the
+Gonum discussion rather than fileing an issue:
```

Comment:
`s/Gonum discussion/Gonum discussion list/` or something similar?
(to the non-native english speaker that I am, it sounded weird, like a word was missing.)

Reply:
Hah! I had it that way before and edited to something else and then didn't revert it properly.

Also spelling of "filing".

Fixed